src/devices/Bmp180/CalibrationData.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Iot.Device.Bmp180
+{
+    internal class CalibrationData
+    {
+        public short AC1 {get;set;}

```

Comment:
Do we need the setters?

Reply:
same answer 

https://github.com/dotnet/iot/pull/184#discussion_r256188500

https://github.com/dotnet/iot/blob/f485b50fae21a044c1ddfa3876426a278de26eaf/src/devices/Bmp280/CalibrationData.cs#L7-L9